src/styles/ProgramsLandingStyles/yoga.less

```diff
@@ -0,0 +1,109 @@
+// <----- CoderYoga ----->
+
```

Comment:
I like how you styled this using divs to separate all your containers and wrapping them correctly to display the awesome design.